test/SdkTests/SdkTests.csproj

```diff
@@ -84,6 +84,7 @@
       <RuntimeVersionToInstall Include="2.1.0" />
       <RuntimeVersionToInstall Include="2.2.5" />
       <RuntimeVersionToInstall Include="3.0.0" />
+      <RuntimeVersionToInstall Include="3.1.0-preview1.19506.1" />
```

Comment:
Can we factor this out into a common property (same version number is hardcoded in GenerateBundledVersions.targets).

Reply:
Good call, I'll do that.